libs/domain/user/overview/overview.component.ts

```diff
@@ -0,0 +1,14 @@
+import { I18nMixin } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { styles } from './overview.styles';
+
+export class OverviewComponent extends I18nMixin(LitElement) {
+  static styles = styles;
+
+  protected override render(): TemplateResult {
+    return html`<oryx-heading
+        ><h3>${this.i18n('account.overview')}</h3></oryx-heading
+      >
+      <div class="content"></div>`;
```

Comment:
We will have to clean it up before release. Can you please leave a todo?

Reply:
Probably we can just remove this component as long as for now it doesn't make any sense.

Reply:
We can leave without empty grey box for sure.